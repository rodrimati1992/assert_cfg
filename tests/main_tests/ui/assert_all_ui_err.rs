assert_cfg::all!{
    feature = "foo",
    feature = "bar",
    feature = "qux",
}

assert_cfg::all!{
    feature = "qux",
}

assert_cfg::all!{
    feature = "bar",
    all(feature = "qux"),
}


fn main(){}