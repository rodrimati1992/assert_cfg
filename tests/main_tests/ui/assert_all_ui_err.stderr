error[E0080]: evaluation of constant value failed
 --> tests/main_tests/ui/assert_all_ui_err.rs:1:1
  |
1 | / assert_cfg::all!{
2 | |     feature = "foo",
3 | |     feature = "bar",
4 | |     feature = "qux",
5 | | }
  | |_^ the evaluated program panicked at '
too few features are enabled, these need to be enabled:
- `feature = "qux"`
', $DIR/tests/main_tests/ui/assert_all_ui_err.rs:1:1
  |
  = note: this error originates in the macro `assert_cfg::all` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0080]: evaluation of constant value failed
 --> tests/main_tests/ui/assert_all_ui_err.rs:7:1
  |
7 | / assert_cfg::all!{
8 | |     feature = "qux",
9 | | }
  | |_^ the evaluated program panicked at '
too few features are enabled, these need to be enabled:
- `feature = "qux"`
', $DIR/tests/main_tests/ui/assert_all_ui_err.rs:7:1
  |
  = note: this error originates in the macro `assert_cfg::all` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0080]: evaluation of constant value failed
  --> tests/main_tests/ui/assert_all_ui_err.rs:11:1
   |
11 | / assert_cfg::all!{
12 | |     feature = "bar",
13 | |     all(feature = "qux"),
14 | | }
   | |_^ the evaluated program panicked at '
too few features are enabled, these need to be enabled:
- `all (feature = "qux")`
', $DIR/tests/main_tests/ui/assert_all_ui_err.rs:11:1
   |
   = note: this error originates in the macro `assert_cfg::all` (in Nightly builds, run with -Z macro-backtrace for more info)
